### PR TITLE
Update MS MARCO doc and passage experiment docs for repo reorganization

### PR DESCRIPTION
**Pull request by MXueguang**
Replication Status:
        Success (No issues)

Environment:
        OS: Ubuntu-18.04.1
        Java: 11.0.7
        Maven: 3.6.0

Results:
        MARCO-passage:
491589963614_ pic_hd
         MARCO-doc:
501590026645_ pic_hd

### CODE REVIEW

**Comment by lintool** [comment]
hey @MXueguang thanks for trying this out.

We've recently updated the repo directory layout: https://github.com/castorini/anserini/pull/1202

That's you! :)
https://github.com/castorini/anserini/pull/1202#issuecomment-629676151

Can you please update the docs with the right paths? The raw collection goes in `collections/`, the index should be built into `indexes/`, runs and logs go into `runs/` and `logs/`, respectively?

Thanks!

**Comment by MXueguang** [comment]
> Can you please update the docs with the right paths? The raw collection goes in `collections/`, the index should be built into `indexes/`, runs and logs go into `runs/` and `logs/`, respectively?

Sure, only msmarco-docs and msmarco-passage? or other experiments as well?

**Comment by lintool** [comment]
msmarco docs and passage to start with... thanks!

**Comment by MXueguang** [comment]
Should the following two parts considered as raw collections as well?

In msmacro-doc
531590075676_ pic_hd

In msmacro-passage
541590075800_ pic

In other words, (for example msmacro-passage) previously we put everything into `msmacro-passage/` but now we will put things to `collections/msmacro-passage`, `logs/msmacro-passage`, `runs/msmacro-passage`, `logs/msmacro-passage` respectively (and only these 4 folders).  Right?

**Comment by lintool** [comment]
> In other words, (for example msmacro-passage) previously we put everything into `msmacro-passage/` but now we will put things to `collections/msmacro-passage`, `logs/msmacro-passage`, `runs/msmacro-passage`, `indexes/msmacro-passage` respectively (and only these 4 folders). Right?

The passage collection should probably be in it's own nested folder, but I don't think we need nested folders for the rest...

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -9,10 +9,12 @@ We're going to use `msmarco-doc/` as the working directory.
 First, we need to download and extract the MS MARCO document dataset:
 
 ```
-mkdir msmarco-doc
-mkdir msmarco-doc/collection
+mkdir collections/msmarco-doc
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:12]: These two:
```
mkdir collections/msmarco-doc
mkdir indexes/msmarco-doc
```

Don't need subdirectories here:
```
mkdir logs
mkdir runs
```

**Pull request merged**